.github/actions/inspect-releaser/action.yml

```
@@ -6,6 +6,10 @@ inputs:
     description: "URL to the artifacts of the releaser workflow"
     required: true
     default: ${{ github.event.workflow_run.artifacts_url }}
+  artifact-names:
+    description: "Names of the artifacts to inspect"
+    required: true
+    default: version.json
```

Name is plural, are we missing an array here?

```suggestion
    default:  '["version.json"]
```